core/combinators/combinators-docs.factor

```diff
@@ -343,16 +343,17 @@ HELP: no-cond
 { $error-description "Thrown by " { $link cond } " if none of the test quotations yield a true value. Some uses of " { $link cond } " include a default case where the test quotation is " { $snippet "[ t ]" } "; such a " { $link cond } " form will never throw this error." } ;
 
 HELP: case
-{ $values { "obj" object } { "assoc" "a sequence of object/word, quotation pairs, with an optional quotation at the end" } }
+{ $values { "obj" object } { "assoc" "an assoc of object/quotation pairs, with an optional quotation at the end" } }
```

Comment:
It's a sequence, not an assoc, because the "optional quotation" isn't a key/value pair at the end:

```factor
15 {
    { 1 [ "one" ] }
    { 2 [ "two" ] }
    [ number>string ]
} case
```